src/Provider/DatabaseMenuProvider.php

```diff
@@ -50,7 +52,17 @@ public function __construct(BuilderInterface $builder, ContainerInterface $conta
      */
     public function get($name, array $options = array())
     {
-        return $this->builder->build($name, $this->container->get('request_stack')->getCurrentRequest());
+        /** @var RequestStack $requestStack */
+        $requestStack = $this->container->get('request_stack');
+        /** @var MatcherInterface $matcher */
+        $matcher = $this->container->get('knp_menu.matcher');
```

Comment:
(flavor) Je zou mij een groot plezier doen door het gebruik van de `container` weg te werken en de request_stack en de matcher in de constructor te injecteren :smiley:

Reply:
Fixed, de matcher heb ik daarnaast ook optioneel gemaakt